mimesis/providers/numbers.py

```diff
@@ -45,6 +48,26 @@ def integers(self, start: int = 0, end: int = 10,
         return self.random.randints(
             length, start, end)
 
+    def complexnums(self, start_real: float = 0, end_real: float = 1,
```

Comment:
Don't you think that it's better to use similar names for methods? You know like floats, integers, complexes.

Reply:
Yes, I was thinking to `complexes()` too, but I wasn't sure it's a correct English word. I will change it.

Reply:
Yeah, me too. I googled it: https://en.m.wiktionary.org/wiki/complex